pallets/nfts/src/features/create_delete_item.rs

```diff
@@ -70,6 +70,15 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 				collection_details.items.saturating_inc();
 
+				let account_balance =
+					AccountBalance::<T, I>::mutate(collection, &mint_to, |balance| -> u32 {
+						balance.saturating_inc();
+						balance.clone()
+					});
+				if account_balance == 1 {
```

Comment:
If the updated balance is equal to 1, it means this is the first item of the account, hence, add the account to `collection_details.item_holders`.